### PR TITLE
Minio

### PR DESCRIPTION
**Pull request by julialongtin**

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -141,6 +142,22 @@ elasticsearch_network_interface=ens3
 poetry run ansible-playbook -i hosts.ini elasticsearch.yml -vv
 ```
 
+### Minio
+
+* In your 'hosts.ini' file, in the `[minio:vars]` section, set 'minio_network_interface'. For example:
```
Comment (jschaul) [inline ansible/README.md]: This should state that specifying a network interface is mandatory (the defaults don't work correctly).

File: ansible/minio.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# Commented out for now. should not depend on hostname.
+#    - role: hostname
+#      tags:
+#        - hostname
+
+# Also commented out. ntp broken?
+#     - role: ansible-role-ntp
+#       tags:
+#         - ntp
```
Comment (jschaul) [inline ansible/minio.yml]: Feel free to remove these comments entirely.

**Comment by tiago-loureiro** [comment]
Any reason why we don't use `hostname` and `ntp` roles here?

**Comment by jschaul** [comment]
> Any reason why we don't use hostname and ntp roles here?

Do we need those? I guess it's orthogonal to minio. Does minio need ntp?

**Pull request merged**